Views/Products/Create.cshtml

```diff
@@ -0,0 +1,60 @@
+@model Bangazon.Models.Product
+
+@{
+    ViewData["Title"] = "PostProduct";
+}
+
+<h2>@ViewData["Title"]</h2>
+
+<form asp-action="Create">
+    <div class="form-horizontal">
+        <h4>Product</h4>
+        <hr />
+        <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+        <div class="form-group">
+            <label asp-for="Name" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <input asp-for="Name" class="form-control" />
+                <span asp-validation-for="Name" class="text-danger" />
```

Comment:
Change all of these self-closing tags to normal tags and validation will work, like so:
`<span asp-validation-for="Name" class="text-danger"></span>`